makexpi.sh

```diff
@@ -101,7 +100,25 @@ fi
 # Prepare packages suitable for uploading to EFF and AMO, respectively.
 [ -d pkg ] || mkdir pkg
 rsync -aL --delete --delete-excluded --exclude /chrome/content/rules src/ pkg/xpi-eff
-cp -a translations/* pkg/xpi-eff/chrome/locale/
+
+# The next lines are for embedded WebExtensions, and can be deleted after the full transition to WebExtensions
```

Comment:
You should probably include a footer indicating the end of the lines that can be deleted after the full transition.